p2p/host/autorelay/timer.go

```diff
@@ -0,0 +1,42 @@
+package autorelay
+
+import (
+	"time"
+
+	"github.com/benbjohnson/clock"
+)
+
+type timer struct {
```

Comment:
Do we need this? It's only used in one spot and it's not hard to call .Stop() yourself

Reply:
It's not hard to call stop, but we need to drain the channel. But only if the timer was running before, otherwise `<-t.C` will block indefinitely.

Reply:
ah I see. Makes sense, thanks